url_creator.py

```diff
@@ -0,0 +1,5 @@
+import os.path
+
+def url_creator(product_id):
```

Comment:
the file name is 'url_creator', the function name should be something like 'create_product_url'